index.html

```diff
@@ -0,0 +1,54 @@
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+  
+    <meta charset="UTF-8" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,200;1,300;1,400;1,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css" />
+    <title>portfolio</title>
+  </head>
+  <body>
+      
+        <div class="container">
+        <div class="section">
+        
+      <div class="mainsection">
+        <div class="maincontainer">
+      <div class="section1">
+        <h2 class="text1">
+          i'm Abel<br />
+          Glad to see you!</h2><br/>
+        
+        <h2 class="text2">
+          I'm a software developer! i can help you build a product, feature or website Look through someof my work and experience! if you like what you see and have you need coded, don't hesitate to contact me.
+        </h2>
+        <div class="let">
+        <p class="text3">LET'S CONNECT</p>
+      </div>
+      </div>
+        </div>
+        <div class=" social_links">
+          <div class="twit-icon"><img src="images/twitter.png" alt="twitter"></div>
+          <div class="git-icon"><img src="images/github.png" alt="github"></div>
+          <div class="ln-icon"><img src="images/Linkedin-icon.png" alt="Linkedin-icon"></div>
+          <div class="m-icon"><img src="images/medium.png" alt="medium"></div>
+          <div class="a-icon"><img src="images/angellist.png" alt="angellist"></div>
+
+        </div>
```

Comment:
- [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Kindly work on that
- [ ] Kindly use flex to display the icons list